WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -170,6 +170,12 @@
                 <% end %>
               <% end %>
 
+              <!-- Show WEC leader a link to eligible voters list. -->
+              <% if current_user.team_leader? Team.wec %>
```

Comment:
I think there's a bit of a philosophical question here regarding if this should be `if can_see_eligible_voters` vs what you've written here. I have a preference for using `can_see_eligible_voters?` again here rather than repeating the "are they the leader of the WEC" logic here, even though it means the Board, WST, and other admins would have a little more clutter to deal with.

Reply:
Yeah, you're totally right that "philosophical" was not the right word here. What I meant is that I think it'll be a tiny bit of maintenance headache to remember to update this if statement if we decide to give anyone besides the leader of the WEC access to this page, because we'll have to remember to update two places.

That said, this is a minor issue, and very easily fixed, so I'm totally ok with what you've done here!

Reply:
So a combination of our both concerns could be fixed with:
```ruby
if current_user.can_see_eligible_voters? && !current_user.can_admin_results?
```
So it would get displayed to everyone with the right access, unless he can access this from the admin section.

Is that any better? =)

Reply:
Ohh, interesting idea! I think I like that if you like it =)